kubectl-cloudflow/Makefile

```diff
@@ -11,9 +11,6 @@ ifeq (release,$(firstword $(MAKECMDGOALS)))
   $(eval $(RELEASE_TAG):;@:) # Make sure arg is not treated as a rule
 endif
 
-
-all: help
```

Comment:
This is removed because it's misleading and repeated. The user can type `make help` to see help info. `make all` shouldn't be printing help messages.